tests/ckeditor/ckeditor.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals describe, it, expect */
+
+'use strict';
+
+bender.amd.require( 'ckeditor', 'ckeditor-core' );
+
+describe( 'getPluginPath()', function( CKEDITOR, core ) {
```

Comment:
Still you would much probably not call `modules.ckeditor` directly, but instead have `var CKEDITOR = modules.ckeditor` on every single test, messing things again. This describe() helper cleans things up by defining this variables once only as its callback parameters.

Also, to make it work right, `var modules` would have to go in the global scope. Maybe not a big deal, but not better than the current solution as well.

Additionally, the proposed gives this sensation of being much similar to AMD (list of requires and modules as params).

I don't know, I still like it a lot and I don't see the big deal.

> Besides, I feel it's too closely coupled with Mocha (which could be avoided thanks to bender.defer).

I don't understand this.